packages/callout-block/src/utils/hasRichTextValue.ts

```diff
@@ -0,0 +1,21 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Use `// eslint-disable-next-line @typescript-eslint/no-explicit-any` for the lines, instead of disable it for the whole file.